applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/prj_release.conf

```diff
@@ -86,3 +86,41 @@ CONFIG_BT_CONN_TX_MAX=4
 CONFIG_SERIAL=n
 CONFIG_CONSOLE=n
 CONFIG_UART_CONSOLE=n
+
+################################################################################
+# DFU configuration
+
+CONFIG_SUIT=y
+CONFIG_SUIT_DFU_CANDIDATE_PROCESSING_MINIMAL=y
+CONFIG_ZCBOR=y
+CONFIG_ZCBOR_CANONICAL=y
+CONFIG_SSF_SUIT_SERVICE_ENABLED=y
+CONFIG_DESKTOP_DFU_MCUMGR_ENABLE=y
+
+CONFIG_MGMT_SUITFU=y
+CONFIG_MCUMGR_SMP_LEGACY_RC_BEHAVIOUR=y
+
+# Enable the Bluetooth MCUmgr transport (unauthenticated).
+CONFIG_MCUMGR=y
+CONFIG_MCUMGR_TRANSPORT_BT=y
+CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN=n
+
+# Allow for large Bluetooth data packets.
+CONFIG_BT_L2CAP_TX_MTU=498
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+
+# Enable the MCUmgr Packet Reassembly feature over Bluetooth and its configuration dependencies.
+# MCUmgr buffer size is optimized to fit one SMP packet divided into five Bluetooth Write Commands,
+# transmitted with the maximum possible MTU value: 498 bytes.
+CONFIG_MCUMGR_TRANSPORT_BT_REASSEMBLY=y
+CONFIG_MCUMGR_TRANSPORT_NETBUF_SIZE=2475
+CONFIG_MCUMGR_TRANSPORT_WORKQUEUE_STACK_SIZE=4096
+
+# Allow a DFU host to retrieve information about the number and size of MCUmgr buffers.
+# The nRF Connect Device Manager uses this information to speed up the DFU.
+CONFIG_MCUMGR_GRP_OS=y
+CONFIG_MCUMGR_GRP_OS_MCUMGR_PARAMS=y
+
+# Extended SUIT commands over USER group
+CONFIG_MGMT_SUITFU_GRP_SUIT=y
```

Comment:
Is this option required for SUIT DFU? Perhaps it could also be selected by the `DESKTOP_DFU_BACKEND_SUIT` option?